dockerfiles/packaging/guice/cassandra/package.sh

```diff
@@ -28,14 +28,22 @@ docker run \
    --volume $PWD:/origin \
    --volume $PWD/dockerfiles/run/guice/cassandra/destination:/cassandra/destination \
    -t james/project -s $SHA1
+docker run \
+   --rm \
+   --volume $PWD/.m2:/root/.m2 \
+   --volume $PWD:/origin \
+   --volume $PWD/dockerfiles/run/guice/cassandra-ldap/destination:/cassandra/destination \
+   -t james/project -s $SHA1
```

Comment:
for a right issue (build files are owned by root), the simplest solution I found to write these file in the cassandra-ldap/destination directory is to redo a build. If you have a better workaround...